client/components/mma/MMAPage.tsx

```diff
@@ -177,6 +177,14 @@ const ContinueMembershipConfirmation = lazy(() =>
 	})),
 );
 
+const SwitchMembershipThankYou = lazy(() =>
```

Comment:
This probably doesn't matter but I wonder should the page components have the same name for the const as the export?